test/test_prototype_builtin_datasets.py

```diff
@@ -76,6 +77,15 @@ def test_decoding(self, dataset, mock_info):
                 f"{sequence_to_str(sorted(undecoded_features), separate_last='and ')} were not decoded."
             )
 
+    @builtin_datasets
+    def test_no_vanilla_tensors(self, dataset, mock_info):
```

Comment:
I would maybe remove this test, not sure if we want to enforce this blindly

Reply:
I think I'll relax it to test if the dataset returns at least one feature. Otherwise the dataset is rather useless for our transforms.